src/Installers/Windows/AspNetCoreModule-Setup/ANCMVS.nuspec

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+    <metadata>
+        <id>VS.Redist.Common.AspNetCoreModule.$ARCH$.$MAJOR$.$MINOR$</id>
+        <version>1.0.0</version>
+        <title>VS.Redist.Common.AspNetCoreModule.$ARCH$.$MAJOR$.$MINOR$</title>
+        <authors>Microsoft</authors>
+        <owners>Microsoft</owners>
+        <license type="expression">$PackageLicenseExpression$</license>
+        <projectUrl>https://github.com/dotnet/aspnetcore</projectUrl>
+        <icon>$PackageIcon$</icon>
+        <requireLicenseAcceptance>true</requireLicenseAcceptance>
+        <description>$MAJOR$.$MINOR$ ASP.NET Core Module ($ARCH$) Windows Installer MSI as a .nupkg for internal Visual Studio build consumption</description>
+        <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+        <serviceable>true</serviceable>
+    </metadata>
+    <files>
+        <file src="$ASPNETCORE_RUNTIME_MSI$" />
+        <!-- Reusing ASPNETCORE_CAB_FILE to add both IIS and IISExpress versions of ANCM. -->
+        <file src="$ASPNETCORE_CAB_FILE$" />
```

Comment:
There's a CAB file? ANCM has always had embedded CABs (ASP.NET had external CABs long ago in 2.0)

Reply:
I've overloading this script to use the the ASPNETCORE_CAB_FILE as the other msi. There are two msis I'd like to insert into this package, and the script only accepted one msi.

I can probably clean this up to use a different variable, but this was easier.